browser-test/src/admin/admin_program_translation.test.ts

```diff
@@ -596,5 +643,54 @@ test.describe('Admin can manage program translations', () => {
         })
       })
     })
+
+    test('External program translations - north star', async ({
```

Comment:
suggestion (non-blocking): Add a single screenshot test for this test so it's easy to see at a glance what the page should look like. I know there's some debate about when we should be adding screenshot tests, and you check for all the fields here so it's fine if you would rather not. But I want to double check that this look right to you and these are the only two fields we should see for external programs? 
<img width="1021" alt="image" src="https://github.com/user-attachments/assets/3d113ed9-4f7a-48bb-ba0e-79c64785c024" />